express-notes-api/server.js

```diff
@@ -0,0 +1,155 @@
+import express from 'express';
+import { readFile, writeFile } from 'node:fs/promises';
+const app = express();
+app.use(express.json());
+
+// errors array
+const errorArr = [
+  { error: 'Id must be a positive integer.' },
+  { error: 'Cannot find note with id.' },
+  { error: 'Content cannot be empty.' },
+  { error: 'An unexpected error occurred.' },
+];
+
+async function parse() {
+  const dataJson = await readFile('data.json', 'utf8');
+  const data = JSON.parse(dataJson);
+  return data;
```

Comment:
You should get used to eliminating these intermediate variables when you can. In this case, by just returning `JSON.parse(...)`.